### PR TITLE
i18n(es): Translated `strapi.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `strapi.mdx`

### CODE REVIEW

File: src/content/docs/es/guides/cms/strapi.mdx [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+  import type Article from '../interfaces/article';
+
+  const articles = await fetchApi<Article[]>({
+    endpoint: 'articles', // el tipo de contenido a obtener
+    wrappedByKey: 'data', // la clave para desempaquetar la respuesta
+  });
+  ---
+  ```
+
+  La llamada a la API solicita datos desde `http://localhost:1337/api/articles` y devuelve `articles`: un array de objetos JSON que representa tus datos:
```
Comment (dreyfus92) [inline src/content/docs/es/guides/cms/strapi.mdx:166]:
```suggestion
  La llamada a la API solicita datos desde `http://localhost:1337/api/articles` y devuelve `articles`: un arreglo de objetos JSON que representa tus datos:
```

File: src/content/docs/es/guides/cms/strapi.mdx [hunk 2]
```diff
@@ -0,0 +1,425 @@
...
+        createdAt: '2023-05-28T13:19:46.421Z',
+        updatedAt: '2023-05-28T13:19:46.421Z',
+        publishedAt: '2023-05-28T13:19:45.826Z'
+      }
+    },
+    // ...
+  ]
+  ```
+
+3. Utilizando los datos del array `articles` devuelto por la API, muestra tus publicaciones de blog de Strapi en una lista. Estas publicaciones estarán vinculadas a sus propias páginas individuales, las cuales crearás en el siguiente paso.
```
Comment (dreyfus92) [inline src/content/docs/es/guides/cms/strapi.mdx:186]:
```suggestion
3. Utilizando los datos del arreglo `articles` devuelto por la API, muestra tus publicaciones de blog de Strapi en una lista. Estas publicaciones estarán vinculadas a sus propias páginas individuales, las cuales crearás en el siguiente paso.
```

File: src/content/docs/es/guides/cms/strapi.mdx [hunk 3]
```diff
@@ -0,0 +1,425 @@
...
+
+2. En la pestaña **Git**, busca la sección **Deploy Hooks**.
+
+3. Proporciona un nombre para tu webhook y la rama en la que deseas desencadenar la compilación. Haz clic en **Add** y copia la URL generada.
+
+##### Agregando un webhook a Strapi
+
+Sigue [la guía de webhooks de Strapi](https://strapi.io/blog/webhooks) para crear un webhook en tu panel de administración de Strapi.
+
+## Official Resources
```
Comment (dreyfus92) [inline src/content/docs/es/guides/cms/strapi.mdx:423]:
```suggestion
## Recursos oficiales
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Buen trabajo @Waxer59, te dejo un par de sugerencias 😁

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for the great work 🚀

**Pull request merged**